Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -724,49 +709,21 @@ class LoggerTests: XCTestCase {
         }
     }
 
-    // MARK: - Tracking Consent
-
-    func testWhenChangingConsentValues_itUploadsOnlyAuthorizedLogs() throws {
-        let consentProvider = ConsentProvider(initialConsent: .pending)
```

Comment:
I'm removing this tests, because in V2 tracking consent will be fully managed by the Core module and transparent to Feature modules. We already have it covered in [`FeatureStorage` tests](https://github.com/DataDog/dd-sdk-ios/blob/e2a961ddc9439e523cf234829bd9a27a4d79451d/Tests/DatadogTests/Datadog/Core/FeatureTests.swift#L21).

I'm doing it now, because since this PR we're no longer running full Core backend in part of Logger tests. Both `FeatureStorage` and `FeatureUpload` are mocked, so we don't have their consent logic available.